Referenced by issue #127976, Actions created via Python cannot be assigned to action constraints
